app/routes/groups_.$groupId.tsx

```diff
@@ -3,20 +3,132 @@ import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
 import { json } from '@remix-run/node';
 
+import { Button } from '~/components/ui/button';
+
+import { Image } from '~/components/ui/images';
+
 export const loader: LoaderFunction = async ({ params }) => {
   const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
   return json({ group });
 };
 
 export default function GroupRoute() {
   const data = useLoaderData<typeof loader>();
-
   return (
-    <>
-      <Link to="/groups">back to groups</Link>
-      <h1>{data.group.name}</h1>
-      <p>{data.group.description}</p>
-    </>
+    <div className="bg-primary text-white flex flex-col items-center">
+      <div className="md:flex md:flex-col md:max-w-screen-xl">
+        <div className="p-2 justify-center md:p-10 md:flex md:flex-row md:border-b-2 md:border-b-white">
+          <div className="m-auto rounded-full h-[300px] w-[280px] justify-center items-center md:m-8">
+            {!data.group.imgUrl && !data.group.imgAlt ? (
+              <Image
+                className="w-full"
+                src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708118888/default-group-photo_xhcpqt.png"
+                alt="Buddies taking a selfie"
+              />
+            ) : (
+              <Image className="w-full" src={data.group.imgUrl} alt={data.group.ImgAlt} />
+            )}
+          </div>
+          <div className="md:my-8 md:mx-10 md:flex md:flex-col md:place-content-between">
+            <div className="flex flex-col md:flex-row md:place-content-between">
+              <div className="m-auto">
+                <h2 className="font-extrabold md:text-5xl md:mb-4">{data.group.name}</h2>
+                <h3 className="md:font-extrabold md:text-2xl md:mb-2">Headline: Group headline here</h3>
```

Comment:
Should we use the group's headline if we already have the group's name?

Reply:
I'm envisioning something like 'All Things Web' as the group name and the headline being 'HTML, CSS, and JavaScript Tutoring.' It looks good aesthetically when the group name is big and bold while the header is slightly smaller. We can use them in the 'preview cards' to allow users to briefly read what groups are about without looking at the full description.